app/src/main/java/com/duckduckgo/app/trackerdetection/model/Tds.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2020 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.app.trackerdetection.model
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity(tableName = "tds")
+data class Tds(
```

Comment:
this class seems confusingly short named, to the point where it's not clear what its purpose is. Even `TdsEntity` would be better i think, though i think `tds` by itself is requiring extra knowledge to understand the acronym.

Reply:
`TdsEntity` already exist, what about... `TdsMetadata` or `TdsInfo`?

Reply:
Changed to TdsMetadata.